test/declarationTests.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-env node, mocha */
```

Comment:
Can you add a `.eslintrc` file to the root of this folder that specifies mocha? This whole project should have the `node` env so that shouldn't be needed here.